dev/requirements.txt

```diff
@@ -13,4 +13,4 @@ wheel
 httpx; python_version >= '3.6'
 selenium; python_version >= '3.6'
 sh<2; python_version >= '3.6'
-tenacity; python_version >= '3.6'
+tenacity<8.4; python_version >= '3.6'
```

Comment:
Instead of using a version < 8.4 we can simply avoid "8.4.0". Hopefully the issue will be fixed in upcoming version.

```
tenacity != 8.4.0
```

Reply:
This way we only update Tenacity when it's really needed, and avoid breaking tests with an unwanted update.

Reply:
C'est parfait comme ça. On mettra à jour explicitement. Ce n'est pas à internet de nous imposer notre emploi du temps :clown_face: